app/views/data_access/retrieval.html.erb

```diff
@@ -78,7 +78,22 @@
                  data-method="post"
                  data-confirm="Are you sure you want to remove this order?  This action cannot be undone."
                  rel="nofollow">Cancel</a>
-              <!-- /ko -->
+            <!-- /ko -->
+            <!-- ko if: dropped_granules -->
+              <p>The following granules will not be processed because they do not support the <strong data-bind="text: method_name"></strong> access method:</p>
+              <div class="granule-list">
+                <div>
```

Comment:
This div doesn't seem to serve any purpose. I'm not sure the div above it does either.